static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -33,10 +33,14 @@ const affixTimeline = () => {
     }
 };
 
+const initFilterCheckbox = () => {
+    const filterKeyEvents = window.location.search.includes('filterKeyEvents=true');
+    document.getElementById("filter-switch").checked = filterKeyEvents;
+}
+
 const createAutoUpdate = () => {
-    if (config.get('page.isLive')) {
-        autoUpdate();
-    }
+    const pollUpdates = config.get('page.isLive')
```

Comment:
we should never call updateBlocks at all if we're on a deadblog

Reply:
@joecowton1

Reply:
This was changed because otherwise we don't load the `blockUpdate()` js if we're on a deadblog - we need the js to listen for events on the button and to send the filter param to the backend.

There might be a better way of doing this though.

Reply:
Renamed `pollUpdates` to `isLiveblog` here: https://github.com/guardian/frontend/pull/24342

Reply:
We need to call update blocks to listen when we filter and un-filter and update the blocks, basically.

Reply:
We have now reverted this to the original approach and moved the button listener out of `autoUpdate.js`